manifests/l/LuaForWindows/rjpcomputing/5.1.5-52/LuaForWindows.rjpcomputing.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: LuaForWindows.rjpcomputing

```

Better to use publisher first
```suggestion
PackageIdentifier: rjpcomputing.LuaForWindows
```

---

When changing the package ID you will also need to change the names of the file and the folder they are in